### PR TITLE
Relax wait between sync

### PR DESCRIPTION
**Pull request by killuazhu**
The default wait time `0` is too aggressive. My team manages a Github Enterprise instance, we had a user who use the default value and cause 200 calls per second against his own repo.

As a default value, no wait is very aggressive.

### CODE REVIEW

File: cmd/git-sync/main.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ var flRoot = flag.String("root", envString("GIT_SYNC_ROOT", envString("HOME", ""
 	"the root directory for git-sync operations, under which --dest will be created")
 var flDest = flag.String("dest", envString("GIT_SYNC_DEST", ""),
 	"the name of (a symlink to) a directory in which to check-out files under --root (defaults to the leaf dir of --repo)")
-var flWait = flag.Float64("wait", envFloat("GIT_SYNC_WAIT", 0),
+var flWait = flag.Float64("wait", envFloat("GIT_SYNC_WAIT", 3),
```
Comment (thockin) [inline cmd/git-sync/main.go:62]: This is a pretty significant behavioral change.  Changing it to 1 second I could maybe buy, but 3 feels like a lot.

I am planning to cut a new major version eventually (I have some big, breaking changes I want to make) and we could discuss this in that context.  Can you file an issue so we can keep track of it?

Also, there are at least 2 e2e test cases that do not set --wait, so they will probably break after this change ("webhook" and "http").

Comment (killuazhu) [reply cmd/git-sync/main.go]: @thockin I created issue https://github.com/kubernetes/git-sync/issues/244 and reduce the wait time to 1 second. Will look into the test cases.

Comment (killuazhu) [reply cmd/git-sync/main.go]: @thockin after updating the default wait time from 3 to 1 second, I'm able to pass all the tests when running `make test` locally.

**Comment by thockin** [comment]
Thanks!  As trivial as this is, I'd like to sit on it until I fork for a new major release.  I will leave this open and merge once I do that.

Thanks.

**Comment by thockin** [comment]
Thanks!  This will target 3.2.0

/lgtm
/approve

**Pull request merged**